cms/jinja2/templates/components/streamfield/headline_figures_block.html

```diff
@@ -1,3 +1,4 @@
+{# TODO: Replace with ONS component once version without a link is available #}
```

Comment:
Can we get a card in place for this, thanks? E.g. card for things like this https://jira.ons.gov.uk/browse/CMS-319, if one already exists do put the link here =)

Reply:
Spoke to @MaciekBaron regarding the card, was informed @janehughes is already working on this! Resolving!